ui/src/containers/ClusterMonitoring.js

```diff
@@ -275,12 +300,14 @@ const makeClusterStatus = (state, props) => {
     value = CLUSTER_STATUS_UP;
     label = intl.messages.cluster_up_and_running;
   }
-
   if (cluster.error) {
     value = CLUSTER_STATUS_DOWN;
     label = intl.messages[cluster.error] || cluster.error;
   }
-
+  if (cluster.prometheusPodStatusMessage) {
```

Comment:
Here you use `prometheusPodStatusMessage ` message to compute the status of the cluster. It make more sense to use the status of the prometheusPod instead. So after calling ApiK8s.getPodInNamespace, store the status in clusterHealth and use it here and when setting ` clusterHealth.error = 'Unknown';` in the saga